### DISCUSSION TITLE
同样的订阅节点openclash访问速度慢

### DISCUSSION THREAD

**Discussion by WangNian007** [vernesong/OpenClash#3503] [General]
同样的订阅节点在路由器用openclash访问推特加载视频很慢，播放也卡顿，但是在本机使用clash for windows 播放很流畅

**Comment by cjlhll** [comment]
同样，楼主解决了吗

**Reply by WangNian007** [reply]
是因为分流规则问题

**Reply by cjlhll** [reply]
不是呢。我都是用的一个节点，speedtest测速的。我想测节点的速度，跟分流没关系。
